tests/test_views_permissions.py

```diff
@@ -168,6 +206,8 @@ def test_permissions_granted_via_groups_are_listed(self):
     def test_permissions_inherited_are_not_listed(self):
         resp = self.app.get('/permissions', headers=self.admin_headers)
         collections = [e for e in resp.json['data']
+                       # top level '/buckets' does not have an id
+                       if e['uri'] != '/buckets'
```

Comment:
syntax error?

Reply:
This is due to getting a KeyError since the top level bucket doesn't have an id